platform/darwin/src/NSDate+MGLAdditions.h

```diff
@@ -7,4 +7,7 @@
 /// Converts from a duration in seconds to a duration object usable in mbgl.
 mbgl::Duration MGLDurationInSeconds(NSTimeInterval duration);
 
+/// Converts from an mbgl duration object to a duration in seconds.
+NSTimeInterval MGLSecondsFromDuration(mbgl::Duration duration);
```

Comment:
Right, took me a while to figure that out, but true. Do you think it's worth adding an assertion or other check here that it remains in seconds in this part of the code?